scripts/triggers-end-to-end-tests/tests.ts

```diff
@@ -362,6 +362,28 @@ describe("import/export end to end", () => {
     expect(aSnap.val()).to.eql("namespace-a");
     expect(bSnap.val()).to.eql("namespace-b");
 
+    // Delete all of the import files
+    for (const f of fs.readdirSync(dbExportPath)) {
+      const fullPath = path.join(dbExportPath, f);
+      await fs.unlinkSync(fullPath);
```

Comment:
nit: Do we still need this in the test? Won't the files be overwritten by the export later anyway?

Reply:
Yeah the idea of unlinking them is to be sure that the export produces them again and that we're not just seeing "leftover" files.